---
layout: archive
title: ""
permalink: /cv/
author_profile: true
redirect_from:
  - /cv
---

<div align="center"> Vinayak, PhD Candidate </div>
<div align="center"> vinayakv[at]seas.upenn.edu | +1-267-918-6247 </div>

Education
=========
* PhD candidate in Materials Science and Engineering 
  * University of Pennsylvania, 2021 - Present
* B.S.(Research), *First Class with Distinction* in Physics 
  * Indian Institute of Science, 2016 - 2020 

Work experience
===============
* Graduate Research Fellow  
  Univeristy of Pennsylvania (2021-Present)
  * _Using MD to predict the 3D chromatin configuration_: Working on a polymer model to predict the change in 3D conformation of nuclear chromatin in response to external stimuli. The model also predicts changes in the epigenetic distribution and therefore the genetic regulation of the simulated region. The model brings out how the underlying physics governs the biological processes and leads to the emergence of large scale structures and funtions from local interactions.
  * _Effect of substrate stiffness on genomic and epigenetic regulation_: Through wet lab experiments, we are trying to figure out how substrate stiffness affects the genetic regulation of non-cancerous and cancerous cells. Which are the prime genes which are up and down regulated to adapt the cell to stiffer and softer conditions, a central question to understand the metastatic cascade.    
* Research Assistant  
  International Centre for Theoretical Sciences, Bangalore (2020-21)  
  * _Data-driven modeling of the Indian Monsoon_: Initially used spectral domain analysis to decipher the effect of sea surface temperature and pressure on the arrival, departure and intensity of the Indian monsoon. Later on moved to a convolutional neural network paradigm to try and figure out the prediction capacities of the sea surface temperatures on the time and intensity of the rainfall. 
* Undergraduate thesis  
  Jawaharlal Nehru Centre for Advanced Scientific Research, Bangalore (2019-2020)
  * _Predicting DNA conductance using ML_: Led the ML part of the model which predicted the conductance of a given DNA strand with a given sequence. The model bypassed the bottleneck DFT step and produced a speedup of a hundred fold with over 80% accuracy. This work appeared in JCIM, ACS.
  * _Exploring the best input atomic features for graph neural networks_: Used different material property predicting graph neural networks to search for the most reliable input atomic representations.  
* Summer visiting undergraduate researcher  
  University of Manchester (2019) and National Centre Radio Astrophysics (2018)
  * Worked on the Square Kilometer Array pipeline to build a streaming algorithm for sifting (UoM).
  * Used novel data processing techniques to study pulsar dynamics (NCRA). 

Skills
===============
* General: Data analysis, Machine learning, Bioinformatics, Molecular dynamics modelling, Scientific computing
* Programming: Python, Bash, R, C, C++, High performance computing, Git, JAX, PyTorch, Tensorflow, LaTeX, MATLAB, Inkscape, Biorender

Service
===============
* Advising
  * Lucas Sant'Anna (Summer student CEMB REU, 2022)
